sdk/confidentialledger/confidential-ledger-rest/CHANGELOG.md

```diff
@@ -7,19 +7,12 @@
 - User defined endpoint
 - User defined endpoint runtimeoptions
 - User defined endpoint modules
-- User defined roles 
-- 
-## 1.1.0-beta.1 (Unreleased)
-
-### Features Added
-
+- User defined roles
 - User List endpoint
 
-### Breaking Changes
```

Comment:
if we don't plan to release it for now, it's better to keep the breaking changes section.